### PR TITLE
Updates for DR7 documentation

### PR DESCRIPTION
**Pull request by geordie666**
Updated to release version for DR7. Includes:

- Latest data model (new random catalog, new Gaia and fiber flux columns)
- Latest status and depth histogram plots
- New more accurate calculations of area using random catalogs for DR7 (and for previous DRs)
- General updates to the description page for area/number of sources/new techniques/zeropoints, etc.
- Updates to publications

### CODE REVIEW

**Comment by geordie666** [comment]
We still need to update the sizes of the `gallery` and `coadd` directories on the `description` page

File: pages/svtips.rst [hunk 1]
```diff
@@ -20,6 +20,8 @@ The Legacy Survey Sky Viewer is generally intuitive, but here are some additiona
 #. If you click on the RA,Dec coordinate readout box, you can enter an RA, Dec, and zoom number that you'd like to view.
 #. If you select one layer and then select a second layer, you can flip back to the previous layer using the space bar.  This is handy to "blink" between two layers, e.g., to look for moving objects.
 
+A full listing of all of the URL patterns that are supported by the viewer is available `here`_.
 
+.. _`here`: http://legacysurvey.org/viewer-dev/urls
```
Comment (weaverba137) [inline pages/svtips.rst]: Should this be a link to `viewer` not `viewer-dev`?

Comment (geordie666) [reply pages/svtips.rst]: Yep, good catch...I changed it to `viewer`

**Comment by weaverba137** [comment]
I also double-checked the coadd directory and got 21 TB, so let's leave that alone.

**Comment by geordie666** [comment]
@weaverba137 in case you were thinking of merging soon, give me 10 minutes to commit a few things and then we should be good to go.

**Comment by weaverba137** [comment]
OK.  Are you documenting the DR6 problem?  Also gallery = 59 MB.

**Comment by geordie666** [comment]
Yes...well I'd already documented it, but I'm adding a link to David's list. I'll also add the gallery size.

**Comment by geordie666** [comment]
OK @weaverba137 , should be good to go. Please review my few changes and merge when ready.

**Comment by weaverba137** [comment]
One more thing: could you please add today's date to the CHANGES.md file?

**Comment by weaverba137** [comment]
OK, last call for any changes.  Otherwise I'll merge very soon.

**Comment by geordie666** [comment]
OK, I've added today's date.

**Pull request merged**